.github/workflows/publish-package-to-npmjs.yml

```diff
@@ -51,27 +54,35 @@ jobs:
       - name: Build package
         run: npm run build-package
       - name: Publish Latest package
-        if: ${{!github.event.inputs.withTag}}
+        if: ${{!github.event.inputs.with_tag}}
         run: npm publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
       - name: Publish Latest package with tag
-        if: ${{github.event.inputs.withTag}}
-        run: npm publish --tag ${{github.event.inputs.withTag}}
+        if: ${{github.event.inputs.with_tag}}
+        run: npm publish --tag ${{github.event.inputs.with_tag}}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
   cdn-deployment:
     needs: [build]
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+
     strategy:
       matrix:
-        environment: [Integ, production]
-    uses: ./.github/workflows/aws-cdn-deployment.yml
-    with:
-      environment: ${{ matrix.environment }}
-      artifact: uid2SDK-${{ (matrix.environment == 'Integ' && 'development') || matrix.environment }}-${{ needs.build.outputs.sdkVersion}}
-      invalidate_paths: '/uid2-sdk-${{ needs.build.outputs.sdkVersion}}.js'
-    secrets:
-      AWS_BUCKET_NAME: ${{ secrets.AWS_BUCKET_NAME }}
-      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      AWS_DISTRIBUTION_ID: ${{ secrets.AWS_DISTRIBUTION_ID }}
+        environment: [integ, production]
+
+    environment: ${{ matrix.environment }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/cdn_deployment_aws
+        with: 
+          environment: ${{ matrix.environment }}
+          artifact: ${{ (matrix.environment == 'integ' && 'development') || matrix.environment }}SecureSignalScript
```

Comment:
This will publish over the SecureSignalScript file - needs to be updated to the original values.